geomstats/riemannian_metric.py

```diff
@@ -187,8 +228,8 @@ def mean(self, points,
             for i in range(n_points):
```

Comment:
you should be able to do that w/o a for loop

Reply:
Yeah I tried, but without success. I'm putting a TODO for now.